inc/class_parser.php

```diff
@@ -1891,4 +1913,84 @@ function encode_url($url)
 
 		return $url;
 	}
+
+	/**
+	 * Determines whether the resulting HTML syntax is acceptable for output, according to the parser's validation policy.
+	 *
+	 * @param string $source The original MyCode.
+	 * @param string $output The output HTML code.
+	 * @return bool|null
+	 */
+	function output_allowed($source, $output)
+	{
+		if($this->output_validation_policy === 0)
+		{
+			return true;
+		}
+		else
+		{
+			$output_valid = $this->validate_output($source, $output);
```

Comment:
Should this be below the next `if` statement?

Reply:
The function call also causes the errors to be logged; its result is not used in any check for policy `1`, but it is for higher values

Reply:
> Should this be below the next `if` statement?

You are correct. @dvz should be corrected.

Reply:
No, I think @dvz explanation is clear.